woocommerce/woocommerce-android
# Issue/10784 blaze analytics part ii

Part of: #10784 


### Description

Ads tracking for all the new events for Blaze campaign creation native flow

### Testing instructions

1. Trigger Blaze creation flow 
2. Follow the table from the tracking section for Blaze i3 PT: pe5sF9-2mb-p2 and check in the Android logcat for the following output: `🔵 Tracked: event_name, Properties: property_name:value ...` to ensure all of the events from the table are fired appropriately.

Note: the last 2 events from the table `_blaze_campaign_creation_success` and `_blaze_campaign_creation_failed` are not included in this PR as the code for firing the campaign creation and handling the success/error result is not ready yet
